_layout/head.html

```diff
@@ -82,18 +82,15 @@
 <div style="display: flex; justify-content: space-between; width:60%; padding-top: 1em; padding-bottom: 0.3em;">
 <a class="waves-effect waves-light btn-large grey" href="https://github.com/JuliaPackaging/BinaryBuilder.jl">Source</a>
 <a class="waves-effect waves-light btn-large grey" href="https://github.com/JuliaPackaging/BinaryBuilder.jl">Get Started</a>
-<a class="waves-effect waves-light btn-large grey" href="https://juliapackaging.github.io/BinaryBuilder.jl/stable">Documentation</a>
+<a class="waves-effect waves-light btn-large grey" href="https://juliapackaging.github.io/BinaryBuilder.jl/dev/">Documentation</a>
  </div>
 
 </div>
 <div class="watdis">
-    <p style="text-align: center; margin: 10px 0 10px 0;">
-    BinaryBuilder lets you build native binaries for C, C++ and Fortran applications and libraries for all major platforms.
-    </p>
```

Comment:
I'd prefer to keep this. It's useful to have a one-line description of what the thing does somewhere in addition to the list of features. I know it may seem a bit redundant, and I'm sure we could improve it, but I think something like it is important.

Reply:
So just add Rust and Go here?

Reply:
I think it would be ok to leave out since we mention it below. BinaryBuilder doesn't add much for Rust and Go, since they already have robust cross compilation stories. I'm worried adding them too prominently will make people dismissive of the project because of that.